src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -40,6 +42,10 @@ public class CommandTestUtil {
     public static final String VALID_GENDER_BOB = "Male";
     public static final String VALID_UNIVERSITY_AMY = "NUS";
     public static final String VALID_UNIVERSITY_BOB = "NTU";
+    public static final String VALID_ID_AMY = "167839";
```

Comment:
This is also minor, but regarding the naming consideration with the same reason as what Deon reviewed, I would suggest to change to `VALID_JOB_ID_AMY`.